src/Database/Schema/Blueprint.php

```diff
@@ -41,6 +44,15 @@ public function addColumn($type, $name, array $parameters = [])
         return $column;
     }
 
+    /**
+     * @param string $key
+     * @param array  $value
+     */
+    public function addIndexSettings(string $key, array $value): void
```

Comment:
Personal preference/consistency/contextual, choose your weapon. I don't feel having `Index` is necessary, in the context of what you're doing (creating/editing an index) I don't think its needed.

```suggestion
    public function addSettings(string $key, array $value): void
```

Reply:
However, an argument could be made that it does add clarity when writing your mapping files 🤔

Reply:
Think I just defeated my own argument

Reply:
I'll leave it up to you

Reply:
Settings has another meaning I think.  I changes this back and forth about 5 times, so sticking now haha